### PR TITLE
CODEOWNERS

### PR DESCRIPTION
**Pull request by jopadi**
change the code owners

### CODE REVIEW

**Comment by jopadi** [comment]
👍

**Comment by sdudoladov** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # global owners
-*        @erthalion @sdudoladov @Jan-M @CyberDem0n @avaczi @FxKu @RafiaSabih
+*        @erthalion @sdudoladov @Jan-M @CyberDem0n @FxKu @jopadi
```
Comment (FxKu) [inline CODEOWNERS:2]:
```suggestion
*        @sdudoladov @Jan-M @CyberDem0n @FxKu @jopadi
```

**Comment by FxKu** [comment]
@jopadi can you update [MAINTAINERS](https://github.com/zalando/postgres-operator/blob/master/MAINTAINERS) file, too?

**Pull request closed without merging**